app/moves/middleware/set-body.single-requests.js

```diff
@@ -3,15 +3,24 @@ const { set } = require('lodash')
 const dateHelpers = require('../../../common/helpers/date')
 
 function setBodySingleRequests(req, res, next) {
-  const { status, sortBy, sortDirection } = req.query
+  const { status } = req.query
+  const useMoveDate = status === 'approved'
+  const {
+    sortBy = useMoveDate ? 'date' : undefined,
+    sortDirection = useMoveDate ? 'asc' : undefined,
+  } = req.query
   const { dateRange } = req.params
   const locations = req.locations
 
```

Comment:
Any reason for a new line here? All the other variables have been, maybe nice to keep them grouped :)